### PR TITLE
Owners can edit dishes

### PR DESCRIPTION
**Pull request by JesperGreen**
PT Story: https://www.pivotaltracker.com/story/show/131704929

Changes proposed in this pull request:
- Owner can edit dishes
- edit button on dish pages for that owner
- Adds reroute to dish show page after update

What I have learned working on this feature: 
- How to add actions to controllers
- How to add routes for actions
- How to create and render a form partial

Ready for review!

### CODE REVIEW

File: features/add_dish.feature [hunk 1]
```diff
@@ -22,7 +22,7 @@ Scenario: I create a dish and put it on a menu
     | Dish description | Delicious pizza      |
     | Price            | 7000kr               |
   And I check the "Lunch" box
-  When I click the "Add a Dish" button
+  When I click the "Submit" button
```
Comment (tochman) [inline features/add_dish.feature]: Change to `And` instead of `When`

File: features/dish_creation.feature [hunk 2]
```diff
@@ -26,7 +26,7 @@ Feature: As a restaurant Owner
       | Allergy Info     | gluten               |
       | Ingredients      | cheese, sauce, crust |
       | Calories         | 10000                |
-    When I click the "Add a Dish" button
+    When I click the "Submit" button
```
Comment (tochman) [inline features/dish_creation.feature]: Change to `And` instead of `When`

File: features/dish_creation.feature [hunk 3]
```diff
@@ -43,7 +43,7 @@ Feature: As a restaurant Owner
       | Dish Name        |                 |
       | Dish description | Delicious pizza |
       | Price            | 7000kr          |
-    When I click the "Add a Dish" button
+    When I click the "Submit" button
```
Comment (tochman) [inline features/dish_creation.feature]: Change to `And` instead of `When`

File: features/dish_creation.feature [hunk 4]
```diff
@@ -53,7 +53,7 @@ Feature: As a restaurant Owner
       | Dish Name        | Pizza   |
       | Dish description |         |
       | Price            | 7000kr  |
-    When I click the "Add a Dish" button
+    When I click the "Submit" button
```
Comment (tochman) [inline features/dish_creation.feature]: Change to `And` instead of `When`

File: features/dish_creation.feature [hunk 5]
```diff
@@ -63,5 +63,5 @@ Feature: As a restaurant Owner
       | Dish Name        | pizza           |
       | Dish description | Delicious pizza |
       | Price            |                 |
-    When I click the "Add a Dish" button
+    When I click the "Submit" button
```
Comment (tochman) [inline features/dish_creation.feature]: Change to `And` instead of `When`

File: features/edit_dish.feature [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    Given I am on the edit dish page for "Kebab"
+    And I fill in "Dish description" with "Kebab med sås"
+    And I click the "Submit" button
+    Then I should be on the dish page for "Kebab"
+
+  Scenario: I should be the only one who can edit my dishes
+    Given the following users exist
+      | name  | email         | role  |
+      | Bosse | bosse@food.se | owner |
+    When I log in as "Bosse"
```
Comment (tochman) [inline features/edit_dish.feature]: Change to `And` instead of `When`

**Review by tochman — CHANGES_REQUESTED** [review]

File: features/add_dish.feature (hunk 1 above)
Comment (JesperGreen) [reply features/add_dish.feature]: Fixed!

File: features/dish_creation.feature (hunk 2 above)
Comment (JesperGreen) [reply features/dish_creation.feature]: Fixed!

File: features/dish_creation.feature (hunk 3 above)
Comment (JesperGreen) [reply features/dish_creation.feature]: Fixed!

File: features/dish_creation.feature (hunk 4 above)
Comment (JesperGreen) [reply features/dish_creation.feature]: Fixed!

File: features/dish_creation.feature (hunk 5 above)
Comment (JesperGreen) [reply features/dish_creation.feature]: Fixed!

File: features/edit_dish.feature (hunk 6 above)
Comment (JesperGreen) [reply features/edit_dish.feature]: Fixed!

**Review by tochman — APPROVED** [review]

**Pull request merged**